content/handbook/security/corporate/end-user-services/laptop-management/laptop-security/encryption/_index.md

```
@@ -0,0 +1,53 @@
+
+## 暗号化の証明
+
+完全ディスク暗号化の証明を提出するには、稼働しているシステムに応じて以下を実施してください。
+
+### Apple macOS
+
+完全ディスク暗号化が有効になっていることの確認画面と、シリアル番号を示す情報の両方を、2 つのウィンドウを横並びで表示した 1 枚のスクリーンショットで撮影してください。
+
+両方の情報は、以下の手順で確認できます。
+
+1. メニューバーで ** > システム設定**をクリックします。
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)